### ISSUE TITLE
DEFOLD:ERROR:EXTENSION: Extension Crash is already registered!\n

### ISSUE THREAD

**Issue by d954mas** [defold/defold#9052]
**Describe the bug (REQUIRED)**
I see some errors in poki dashboard.

```
"DEFOLD:ERROR:EXTENSION: Extension Crash is already registered!\n"
```
<1%

**To Reproduce (REQUIRED)**
No ideas(

**Expected behavior (REQUIRED)**
No error? i am not sure what is happened in real game

**Defold version (REQUIRED):**
  - Version 1.8.1

**Platforms (REQUIRED):**
 - Platforms: html5
 
**Minimal repro case project (OPTIONAL):**

**Logs (OPTIONAL):**

**Workaround (OPTIONAL):**

**Screenshots (OPTIONAL):**

**Additional context (OPTIONAL):**

**Comment by britzl** [comment]
@d954mas are you using sys.reboot()?

**Comment by ekharkunov** [comment]
@d954mas It looks like you have double initialization of engine.
Do you have custom engine_template.html? Can it be situation when `Module._preloadAndCallMain()` called twice? Do you have any changes to default `dmloader.js`?

**Comment by d954mas** [comment]
> are you using sys.reboot()?

no

**Comment by d954mas** [comment]
> @d954mas It looks like you have double initialization of engine. Do you have custom engine_template.html? Can it be situation when `Module._preloadAndCallMain()` called twice? Do you have any changes to default `dmloader.js`?

Yes i have custom html template. No i don't make changed in dmloader.js

[engine_template.txt](https://github.com/user-attachments/files/15758338/engine_template.txt)

**Comment by JCash** [comment]
Do you have any extra symbols specified in your extensions `ext.manifest` files (or your app manifest)?

**Comment by JCash** [comment]
Wait, I do remember now.
It was an issue on the build server, where it accidentally added the symbol name twice.
There was a fix some 2 months ago though, so this build is probably _not_ 1.8.1 as reported? 🤔 

(the fix https://github.com/defold/extender/commit/571d148a283f288813d22dcb7220cea0cadb4918)

**Comment by JCash** [comment]
The recommended action is to rebuild the game with a newer server.
I'm closing the issue, but feel free to reopen if the problem persists.

**Comment by d954mas** [comment]
I also see that error
console.error: ["ERROR:DLIB: dmLog already initialized"]
2 users,

Desktop
Safari 17.5.0
Desktop
Safari 16.0.0
